src/components/PasswordToggle/PasswordToggle.tsx

```diff
@@ -4,11 +4,16 @@ import type { InputHTMLAttributes, ReactNode } from "react";
 
 import Button from "../Button";
 import Field from "../Field";
-import Label from "../Label";
+import VanillaLabel from "../Label";
 
 import type { ClassName, PropsWithSpread } from "types";
 import { useId } from "hooks";
 
+export enum Label {
```

Comment:
Oh, this makes it confusing. `Label` is a component and also an enum. And having it in title case `Label` for enum, makes it look like it's a component. 

But I don't have any better suggestion for now.

Reply:
Yeah good point, we could rename these to something like `LabelStrings` or `Text`.